src/python/pygrackle/utilities/model_tests.py

```diff
@@ -11,11 +11,18 @@
 # software.
 ########################################################################
 
+import argparse
 import itertools
 import os
+import sys
 
 from pygrackle import chemistry_data
 
+if sys.version_info < (3, 9):
+    from functools import lru_cache as cache
+else:
+    from functools import cache
+
```

Comment:
Python 3.8 is fairly ancient and not even supported by yt anymore. I think we should drop 3.8 support and forget the version checking.